logstash-core/lib/logstash/webserver.rb

```diff
@@ -30,8 +31,10 @@ def initialize(logger, options={})
                                      # https://github.com/puma/puma/pull/640 for mode internal details in PUMA.
                                      :queue_requests => false
       })
-      @status      = nil
 
+      logger.terminal("Binding Logstash WebAPI to tcp://#{http_host}:#{http_port}")
```

Comment:
Can we change to  `Binding Logstash HTTP server to #{http_host}:#{http_port}"` We could be adding more than web API in the future..